src/components/organisms/KeyboardShortcuts/KeyboardShortcuts.tsx

```diff
@@ -0,0 +1,131 @@
+import {Button, Modal, Typography} from 'antd';
+
+import hotkeys from '@constants/hotkeys';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {closeKeyboardShortcutsModal} from '@redux/reducers/ui';
+
+import {defineHotkey} from '@utils/defineHotkey';
+
+import Keyboard from '@assets/Keyboard.svg';
+
+import BoardKeys from './BoardKeys';
+
+import * as S from './styled';
+
+const {Text} = Typography;
+
+const shortcutsNavigation = [
+  {name: 'Navigate Back', value: defineHotkey(hotkeys.SELECT_FROM_HISTORY_BACK)},
```

Comment:
I have one 'API' remark but I think it might be more work than is worth the effort. Hotkeys are something we'll regularly want to add so the design should make it easy to extend without having to go across multiple files.

```
export type Hotkey = keyof typeof HOTKEYS;

export const HOTKEYS = createHotkeys({
  SELECT_FROM_HISTORY_BACK: {
    name: 'test',
    key: 'cmd + h',
    category: 'tool',
  },
});

type HotkeyConf = {
  name: string;
  key: string;
  category: 'navigation' | 'tool';
};

function createHotkeys<TName extends string>(config: Record<TName, HotkeyConf>) {
  return config;
}
```

Then the other pages should be dynamic to avoid having to edit multiple files when adding a key:

```
const shortcutsNav = entries(HOTKEYS)
  .filter(([_, hotkey]) => {
    return hotkey.category === 'navigation';
  })
  .map(([id, hotkey]) => {
    return {name: hotkey.name, value: defineHotkey(hotkey.key)};
  });
```